frontend/components/commons/forms/questions-form/QuestionsForm.component.vue

```diff
@@ -27,7 +27,7 @@
           v-if="input.component_type === COMPONENT_TYPE.FREE_TEXT"
           :title="input.question"
           :placeholder="input.placeholder"
-          v-model="input.options[0].text"
+          v-model="input.options[0].value_in_api"
```

Comment:
Just a comment, I feel a bit confused with value_in_api name in a dynamic frontend v-model

Reply:
I replace it by `value`